Proposal for text addressing Bucket Types and other common parameter (#56)
ietf-wg-mediaman/6838bis: draft-ietf-mediaman-6838bis.md

There is no defined syntax for parameter values; therefore, it needs to be specified upon registration. Additionally, some transports impose restrictions on parameter value syntax, so care needs be taken to limit the use of potentially problematic syntaxes; for example, binary valued parameters, while permitted in some protocols, are best avoided.

> There exist parameters that are being reused across multiple media types definitions to provide a common functionality. The 'Codecs' and 'Profiles' Parameters for "Bucket" Media Types {{!RFC6381}} are one such example used to indicate which media codecs and their complexity are represented inside the data object identified by the media type. Another set of common parameters are used by RTP payload formats, see {{!RFC4855}}, and {{!RFC8851}}.

Reviewer: The most well known "common" parameter is probably "charset" from the text/ top level type.

> There exist parameters that are being reused across multiple media types definitions to provide a common functionality. The 'Codecs' and 'Profiles' Parameters for "Bucket" Media Types {{!RFC6381}} are one such example used to indicate which media codecs and their complexity are represented inside the data object identified by the media type. Another set of common parameters are used by RTP payload formats, see {{!RFC4855}}, and {{!RFC8851}}.

Reviewer 2: ```suggestion
Some parameters are reused across multiple media type definitions to provide common functionality. For example, the 'Codecs' and 'Profiles' Parameters for "Bucket" Media Types {{!RFC6381}} indicate which media codecs and their complexity are represented inside the data object identified by the media type. Another example is used with RTP payload formats: see {{!RFC4855}}, and {{!RFC8851}}.
```

Editorial nits above. I don't know what "indicate which media codecs and their complexity are represented inside the data object identified by the media type" means, though.

Reviewer: That should probably be replaced with "identify media codecs used inside the container and their parameters" - registration procedures shouldn't be concerned with what the parameters indicate.
